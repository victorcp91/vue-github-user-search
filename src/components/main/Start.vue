<template>
  <div>
    <logo></logo>
    <search></search>
  </div>  
</template>

<script>
import logo from './components/logo';
import search from './components/Search';

export default {
  components: {
    logo,
    search
  }
}
    
</script>

<style lang="scss" scoped>

</style>
