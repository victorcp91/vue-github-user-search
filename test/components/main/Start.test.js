import { shallow } from 'vue-test-utils';
import Start from '../../../src/components/main/Start';

describe('Start.test.js', () => {
    let wrapper;

    beforeEach(() => {
        wrapper = shallow(Start);
    });

    it('should match html structure', () => {
        expect(wrapper.element).toMatchSnapshot();
    });
});