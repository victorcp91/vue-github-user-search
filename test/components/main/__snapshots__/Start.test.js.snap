// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Start.test.js should match html structure 1`] = `
<div>
  <!---->
   
  <!---->
</div>
`;
